### PR TITLE
Various bugfixes. Pending new feature updates.

### PR DESCRIPTION
**Pull request by MacMillan-M**
[Bugfix] Readded Admin Slots
Admin-1
Pilot: Admin-1-1
Pw: See pinned message in S3 Dev DM.

[Feature] Still pending the delay/resume restart command to be secured within the admin slot.

Kept Airboss and ATIS in. They didnt modify the radio menu too much. We can still disable if desired.

[Bugfix] Re-edited the options to try and see if labels were fixed.

### CODE REVIEW

**Comment by MacMillan-M** [comment]
Lets figure out the add for "more" in the radio submenu before pushing this. 

That or we condense the Airboss, ATIS and Settings "USERNAME" menu into a Mission Features radio menu. 

Ideally, we also take a look at securing that delay/resume restart in the Moose and CavUtils lua files to be tied only to the admin slot.

File: l10n/DEFAULT/CTLD.lua [hunk 1]
```diff
@@ -142,7 +142,7 @@ ctld.JTAC_smokeOn_BLUE = true -- enables marking of target with smoke for BLUE f
 ctld.JTAC_smokeColour_RED = 4 -- RED side smoke colour -- Green = 0 , Red = 1, White = 2, Orange = 3, Blue = 4
 ctld.JTAC_smokeColour_BLUE = 1 -- BLUE side smoke colour -- Green = 0 , Red = 1, White = 2, Orange = 3, Blue = 4
 
-ctld.JTAC_jtacStatusF10 = true -- enables F10 JTAC Status menu
+ctld.JTAC_jtacStatusF10 = false -- enables F10 JTAC Status menu
```
Comment (pingoocoder) [inline l10n/DEFAULT/CTLD.lua:145]: Why are you disabling this?

File: l10n/DEFAULT/CTLD.lua [hunk 2]
```diff
@@ -209,8 +209,6 @@ ctld.dropOffZones = {
 	{ "dropzone11", "green", 2 },
 	{ "dropzone12", "green", 2 },
 	{ "dropzone13", "green", 2 },
-	{ "dropzone14", "green", 2 },
```
Comment (pingoocoder) [inline l10n/DEFAULT/CTLD.lua:212]: What does this change do?

File: l10n/DEFAULT/CavCaller.lua [hunk 3]
```diff
@@ -0,0 +1,11 @@
+-- Function to call in the various functions defined in CavUtils, without having to call the entirerity of CavUtils.
```
Comment (pingoocoder) [inline l10n/DEFAULT/CavCaller.lua:1]: entirety*

File: l10n/DEFAULT/CavCaller.lua [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+-- Disable player settings via MOOSE
+_SETTINGS:SetPlayerMenuOff()
+
+-- Admin Menu Tester, REMOVE BEFORE PUSH TO SERVER
+--local DemoMenu = CavUtils.AddAdminforAll()
+-- Adds in the admin menu via MOOSE function
+-- Sets the admin1 variable to the group "Admin-1"
+local admin1 = GROUP:FindByName('Admin-1')
+local MenuAdd = CavUtils.AddAdminMenu(admin1)
```
Comment (pingoocoder) [inline l10n/DEFAULT/CavCaller.lua:11]: From testing, I'm not seeing this option menu in the admin slot. Is this suppose to be working in this PR? Also line 6, were you suppose to remove something?

Comment (pingoocoder) [reply l10n/DEFAULT/CavCaller.lua:11]: We should just disable this functionality until we can get the admin menu working.

File: l10n/DEFAULT/Moose_10_05_21.lua [hunk 5]
```diff
@@ -8882,7 +8882,7 @@ end
 -- @field #SETTINGS
 SETTINGS = {
   ClassName = "SETTINGS",
-  ShowPlayerMenu = true,
+  ShowPlayerMenu = false,
```
Comment (pingoocoder) [inline l10n/DEFAULT/Moose_10_05_21.lua:8885]: What does this do?

**Comment by pingoocoder** [comment]
> Lets figure out the add for "more" in the radio submenu before pushing this.
> 
> That or we condense the Airboss, ATIS and Settings "USERNAME" menu into a Mission Features radio menu.
> 
> Ideally, we also take a look at securing that delay/resume restart in the Moose and CavUtils lua files to be tied only to the admin slot.

I'm not seeing the CTLD option being fixed in the UH-60, may you confirm?

image

File: l10n/DEFAULT/CTLD.lua (hunk 1 above)
Comment (MacMillan-M) [reply l10n/DEFAULT/CTLD.lua:145]: To clean up the support menu. The JTAC status refers to dropped crates from CTLD. We dont use this generally.

File: l10n/DEFAULT/CTLD.lua (hunk 2 above)
Comment (MacMillan-M) [reply l10n/DEFAULT/CTLD.lua:212]: For your drop points in CTLD. It defines where you can drop off the crates and will be recognized. We need to add in the dropzone 14 and 15 to include in the new dropzones at objectives. However, this broke CTLD last itme so it should be commented out/removed until we can figure out the fix.

File: l10n/DEFAULT/CavCaller.lua (hunk 3 above)
Comment (MacMillan-M) [reply l10n/DEFAULT/CavCaller.lua:1]: Will fix for next update.

File: l10n/DEFAULT/Moose_10_05_21.lua (hunk 5 above)
Comment (MacMillan-M) [reply l10n/DEFAULT/Moose_10_05_21.lua:8885]: Part of the settings removal thing. Its a two parter, this and the CavCaller file.

File: l10n/DEFAULT/CavCaller.lua (hunk 4 above)
Comment (MacMillan-M) [reply l10n/DEFAULT/CavCaller.lua:11]: No not working in current version. Need to troubleshoot more. Line 6 was commented out or shouldve been. Was just a tester function to add to the coalition, but its a bug i need to fix.

Comment (pingoocoder) [reply l10n/DEFAULT/CavCaller.lua:11]: Shouldn't we comment out line 11 too then?

Comment (MacMillan-M) [reply l10n/DEFAULT/CavCaller.lua:11]: Ive left line 11 in, will continue to modify. We can comment it out but it looks harmless for now. I think its something wrong with my AddAdminMenu function in CavUtils. Need to look more into the moose scripting info.

**Pull request merged**